packages/gatsby-plugin-treat-example/babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Wouldn't need this in a regular `gatsby` project but I had to override the root `babel.config.js`